### PR TITLE
Vk: add cull mode and front face

### PR DESCRIPTION
**Pull request by raven02**

### CODE REVIEW

**Comment by kd-11** [comment]
@raven02 You need to consider the GL_FRONT_FACE setting as well. Back and front face description will be changed by it.

**Comment by raven02** [comment]
Just done it . Pulling the commit.

**Comment by kd-11** [comment]
Look ok now

File: rpcs3/Emu/RSX/GL/GLGSRender.cpp [hunk 1]
```diff
@@ -464,27 +442,23 @@ void nv4097_clear_surface(u32 arg, GLGSRender* renderer)
...
-};
-
 bool GLGSRender::do_method(u32 cmd, u32 arg)
 {
-	auto found = g_gl_method_tbl.find(cmd);
-
-	if (found == g_gl_method_tbl.end())
-	{
+	switch (cmd)
+	{
```
Comment (11DH11) [inline rpcs3/Emu/RSX/GL/GLGSRender.cpp]: Do not do it. Place back unordered_map, because much more commands would be overridden in near time

Comment (raven02) [inline rpcs3/Emu/RSX/GL/GLGSRender.cpp]: Alright .I will undo this part.

**Pull request merged**